clients/lsp-clangd.el

```diff
@@ -246,7 +249,7 @@ returned to avoid that the echo area grows uncomfortably."
                                  "textDocument/switchSourceHeader"
                                  (lsp--text-document-identifier))))
             ))
-      ;; TODO use lsp-goto-location instead
+      ;; TODO maybe worth using lsp-goto-location instead
```

Comment:
You cannot and shouldn't use `lsp-goto-location`, because that expects an `LspLocation` with line:col information, which is not what `clangd` returns here (it would also not make sense for it to do that).

Reply:
Thank for addressing my TODOs. I will delete the comment.

Reply:
Are there still any ToDos remaining?